app/models/instructor.rb

```diff
@@ -44,12 +44,14 @@ def self.get_user_list(user)
       participants << course.get_participants
     end
     assignments = Assignment.where(instructor_id: user.id)
-    assignments.each do |assignment|
+    #Line 48 implements eager loading using 'includes' method. For detail, visit: http://tiny.cc/9aomuz
+    assignments.includes([:participants]).each do |assignment|
       participants << assignment.participants
     end
     participants.each do |p_s|
```

Comment:
refactor p_s to participant. variable name is unclear

Reply:
The only change we made in line 5 was to add 

.includes([:participants]) + comment

other than this, all the code is as it was before. I am assuming it should be fine as it was already there in the beta branch before we even started working.